### DISCUSSION TITLE
documentación semana #1

### DISCUSSION THREAD

**Discussion by JhonFre7** [Launch-X-Latam/OnBoarding#11] [Q&A]
donde se puede ver toda la documentación que vamos a necesitar?

**Comment by JosafatJimenezB** [comment]
Las practicas que iremos realizando las colocara el mission commander en un repositorio poco a poco, el cual explicara primero en una sesion

saludos!

**Comment by Capirossi** [comment]
No entiendo como es el formato de estudio.
Llevo dos dias buscando informacion de como hacerlo.
Dejo mi vacante disponible

**Reply by JhonFre7** [reply]
Ya viste el live de hoy?

**Reply by devonent** [reply]
Hasta ahora, tengo entendido que consistirá en asistir a los lives en Teams los días miércoles y viernes, y durante el desarrollo se nos estarán asignando algunas tareas que deberemos de realizar (creo que aún no mencionan más características sobre ello). El resto de los conocimientos se deberán adquirir de forma autodidacta.

**Reply by 3nriquecastro** [reply]
Hasta ahora solo me metieron a uno de cyberseguridad ,solo es ese?

**Comment by gumodi35** [comment]
No es un formato de tradicional como se maneja aquí, pero básicamente es asistir a las seciones o verlas on demand y cumplir con las actividades, la mayor parte del programa es ser autodidacta, en la comunidad puedes ir comentando tus dudas
